### ISSUE TITLE
[Question] Scan does not compare file against database? No duplicate detected

### ISSUE THREAD

**Issue by jeffward01** [0x90d/videoduplicatefinder#349]
**Environment**
 - OS: Windows 10 PRO

**Describe the bug**
I am not sure if this is a bug - So i am asking a question.

**Scenerio**
* I tell VDF to **ONLY** scan my library directory at `root/Library/` -  all of these files are now added to the VDF database.  (158,587 files)
* I then take a video from `root/Library/` and copy it to `root/_testDuplicate/`
* I then tell VDF to **ONLY** scan `root/_testDuplicate/`
* VDF Does not find a duplicate

Is this a bug?

This is the log:

![image](https://user-images.githubusercontent.com/12589359/198733720-e98d667b-911e-43c2-aeeb-c8ba9f56ba43.png)

**Expected behavior**
I would expect that when VDF scans **only** `root/_testDuplicate/` that it would compare the scanned result files against the existing database of files (in this case from the files that were previously scanned from `root/Library/`

* **Expected Result:** I would expect (1) duplicate would be found
* **Real Result:** No duplicates are found

**Comment by 0x90d** [comment]
This is expected. VDF only includes files from the database which are part of the "includes" list.

**Comment by jeffward01** [comment]
> This is expected. VDF only includes files from the database which are part of the "includes" list.

Thanks for this. 

I find that this is a bit cumbersome and forces the user to wait long periods of scanning for large libraries.

Please correct me if I am wrong for the below scerenio

**Scenerio**
* I tell VDF to **ONLY** scan my library directory at `root/Library/` -  all of these files are now added to the VDF database.  (158,587 files)
* I then take a video from `root/Library/` and copy it to `root/_testDuplicate/` y/` || Total videos now are 158,588
* I then tell VDF to scan **BOTH**  `root/_testDuplicate/` and the previous  `root/Library/`
* I add **new 2,000 videos to the root library. **   `root/Library/` || Total videos now are 160,588
* VDF finds a duplicate
* VDF **rescans  the root library to locate the new files** files in some capacity?  

**The issue**
Basically I have experienced this to be a tedious operation.  For example:

* I want to check my  `root/_testDuplicate/` against the current database in its entirety.  (Not scan new files)
* I only care about if the files in `root/_testDuplicate/` are duplicated within `root/Library/` as of the last scan. 
    *  Based on the 'last scan' the database has 158,587 files, 
    *  I am aware that I added 2,000 new files to `root/Library/` -- however I do not want to scan these (it takes a long time)
    * I am forced to 're-scan' the new files.


Is there any way around this?

**Comment by 0x90d** [comment]
On new build you can check in settings if you want to scan against your entire database.

**Comment by jeffward01** [comment]
@0x90d Hello!

So that means I can select:

* Include Scan Directory: `root/_testDuplicate/`
* Then set the flag for 'Scan against database'?

Thats pretty cool, that will be extremely helpful!

I think we can close this ticket out now.

**Last question**

When do you expect the new build out? No rush at all, just want to have an idea if its days, weeks, months.  They are all fine for me <3 

Thanks for everything
